src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -211,6 +212,45 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             }
         }
 
+        [Fact]
+        public async Task PostAsync_Post_ChannelBinding_ConfiguredCorrectly()
+        {
+            var content = new ChannelBindingAwareContent("Test contest");
+            using (var client = new HttpClient())
+            using (HttpResponseMessage response = await client.PostAsync(HttpTestServers.SecureRemoteEchoServer, content))
+            {
+                // Validate status
```

Comment:
You're triggering my OCD.  There needs to be periods at the end of the these comments in these files.  That is the standard for System.Net.